Sources/SourceKitLSP/Swift/DocumentFormatting.swift

```diff
@@ -15,6 +15,7 @@ import Foundation
 package import LanguageServerProtocol
 import LanguageServerProtocolExtensions
 import SKLogging
+import SKUtilities
 import SwiftExtensions
 import SwiftParser
 import SwiftSyntax
```

Comment:
@ahoppen I noticed there has been 2 import changes to this file that i had to merge from main.
Just checking that this looks good?
First was addition of `LanguageServerProtocolExtensions` IIRC, the newer one was addition of `SwiftExtensions`.
I also added `SKUtilities`.